config/routes.rb

```diff
@@ -1,4 +1,8 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  devise_for :users, controllers: { registrations: 'users/registrations' } # Use your custom registrations controller
+
```

Comment:
- [ ] Also, I could not possibly hit your login and registration routes, even with the custom and the default routes, It keeps throwing an error. Please take a look and try to address it. 
- [ ] Once the routes are working fine, Please make sure to follow the given wireframe. We need to follow the wireframe closely. 
### The error I face.
![Screen Shot 2023-10-13 at 11 36 08 PM](https://github.com/mosidrum/blog-app/assets/69077061/16cf6174-1757-43e9-946f-1021d05556d2)